### PR TITLE
Add config file for Read the Docs & fix rending of included pages

### LINKED ISSUE #25: Read the docs build is failing
* Django DataDog Logger version: n/a
* Python version: n/a
* Operating System: n/a

### Description

https://readthedocs.org/projects/django-datadog-logger/builds/

Looks like it's missing the [RTD config file](https://docs.readthedocs.io/en/stable/config-file/index.html):

```
Error
No default configuration file found at repository's root.
```

### PR DESCRIPTION
**Pull request by browniebroke**
Fix #25

### CODE REVIEW

**Comment by browniebroke** [comment]
Nice. Do you want to send a PR to my fork so I merge it into my branch? Would give you credits for the commits... Alternatively, I'll let you send a PR to this repo and close mine.

File: docs/conf.py [hunk 1]
```diff
@@ -38,10 +38,7 @@
 templates_path = ["_templates"]
 
 # The suffix(es) of source filenames.
-# You can specify multiple suffix as a list of string:
-#
-# source_suffix = ['.rst', '.md']
-source_suffix = ".md"
```
Comment (browniebroke) [inline docs/conf.py:44]: The curent docs source files are RST, but this line was militing to only looks for markdown files. Additionaly, some of these files includes the markdown from the repo root:

https://github.com/namespace-ee/django-datadog-logger/blob/3870d2c1d71240cd6c795e45b8153644c63a3f75/docs/readme.rst#L1

Resulting in invalid rendering:

image

https://browniebroke-django-datadog-logger--1.org.readthedocs.build/en/1/readme.html

**Comment by jacekbj** [comment]
@browniebroke Go ahead, I don't mind.

**Comment by browniebroke** [comment]
I've addeed`myst-parser`: it enables markdown files in the Sphinx doc that include other markdown files. I didn't try to convert other RestructuredText files, but I can do it if you like.

File: docs/conf.py (hunk 1 above)
Comment (lnagel) [reply docs/conf.py:44]: They indeed were rst, but I've started to convert them to markdown in main already.

**Pull request merged**